tests/integration/test_bincount.py

```diff
@@ -48,6 +49,30 @@ def test_bincount_weights(dtype):
     assert allclose(out_np, out_num)
 
 
+@pytest.mark.parametrize("dtype", DTYPES)
+def test_bincount_high_bins(dtype):
+    v_num = num.array([0, LARGE_NUM_BINS], dtype=dtype)
+
+    v_np = v_num.__array__()
+
+    out_np = np.bincount(v_np)
+    out_num = num.bincount(v_num)
+    assert num.array_equal(out_np, out_num)
+
+
+@pytest.mark.parametrize("dtype", DTYPES)
+def test_bincount_weights_high_bins(dtype):
+    v_num = num.array([0, LARGE_NUM_BINS], dtype=dtype)
+    w_num = num.random.randn(2)
+
+    v_np = v_num.__array__()
+    w_np = w_num.__array__()
+
+    out_np = np.bincount(v_np, weights=w_np)
+    out_num = num.bincount(v_num, weights=w_num)
```

Comment:
```suggestion
    v_num = num.array([0, LARGE_NUM_BINS], dtype=dtype)
    w_num = num.random.randn(2)
    out_num = num.bincount(v_num, weights=w_num)
    
    v_np = v_num.__array__()
    w_np = w_num.__array__()
    out_np = np.bincount(v_np, weights=w_np)
```
This is is a very minor suggestion, but a grouping like this reads nicely as:

* prepare cn value
* prepare np value
* compare cn and np values

Reply:
Done, cleaned up the other tests in the file as well.